src/app/bootstrap/index.ts

```diff
@@ -86,6 +86,4 @@ export const bootstrap = async () => {
       return new Error("Expected BTC-currency default wallet")
     }
   }
-
-  return true
```

Comment:
if this is a fix please add return type to the method

Reply:
the method was not explicitly typed